Modules/Core/Transform/include/itkMatrixOffsetTransformBase.hxx

```diff
@@ -446,7 +447,7 @@ typename MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutp
   InverseTransformBasePointer
   MatrixOffsetTransformBase<TParametersValueType, NInputDimensions, NOutputDimensions>::GetInverseTransform() const
 {
-  Pointer inv = New();
+  typename InverseTransformType::Pointer inv = InverseTransformType::New();
```

Comment:
(Minor issue) Please consider using `auto` here, following pull request https://github.com/InsightSoftwareConsortium/ITK/pull/2826 "STYLE: Use auto for declaration of variables initialized by `New()`"

Reply:
I have now amended it using auto.

Reply:
Thanks @josempozo !